tests/integration/examples/discord/steps/trainer_mlflow.py

```diff
@@ -0,0 +1,46 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
```

Comment:
```suggestion
#  Copyright (c) ZenML GmbH 2023. All Rights Reserved.
```

Reply:
okay will do it later, interested to know...did you run some command?